% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/wrapper_functions.R
\name{err_catch}
\alias{err_catch}
\title{Error catcher}
\usage{
err_catch(f_expression = quote())
}
\arguments{
\item{f_expression}{Expression to be evaluated}
}
\value{
Error
}
\description{
Catches errors directly to stderr and continues execution.
'capture.output' is used to get the output of 'try' expression, whereas 'try' itself is used to evaluate whether or not 'expression' works
}
\examples{
err_catch()
}
